fdbserver/DataDistribution.actor.cpp

```diff
@@ -916,7 +935,7 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 					auto& serverTeams = servers[j]->teams;
 					for( int k = 0; k < serverTeams.size(); k++ ) {
 						auto &testTeam = serverTeams[k]->getServerIDs();
-						bool allInTeam = true;
+						bool allInTeam = true; // Does all servers in testTeam belong to the healthy servers, extracted from current badTeam
```

Comment:
For the team that is too big (too many servers), we will find a subset of servers in the team as the new healthy team) so that moving data to the subset won't cause much overhead in data movement.